report/main.Rmd

```diff
@@ -0,0 +1,313 @@
+
+```{r correlation, echo=FALSE, fig.align='center', fig.height=3, fig.width=3, fig.pos="H"}
+plot_corr_plot(beverage_sales)
+```
+
+
+
+### 3.3 Missing Values and Data Imbalance
+
+The data has some missing data. It is important to identify what kind of missing data exists within a dataset to better understand how to handle missingness during formal analysis. It appears that the missing data qualifies as missing not at random (MNAR), meaning that the probability of any given observation being missing varies for unidentified reasons. It is also important to note from the observations counts whether or not the data appears to be balanced since imbalanced data can hinder model accuracy. Balance between sites appears to be reasonable, where some imbalance is present between interventions. Namely, the 'follow', 'wash', and 'wash2' levels are imbalanced when compared to the other interventions (see figure in appendix).
+
+
+
+### 3.4 Time Series Plot 
+
+The following stacked line plot represents the sales time series of zero-calorie and sugary beverages across different sites. Each line represents the sales trajectory of one beverage type—green for zero-calorie and blue for sugary drinks. The x-axis represents time (in days), and the y-axis represents the sales volume. Dashed vertical lines indicate the start of different interventions, labelled as dismes (discount & messaging), dis (only discount), cal (calorie content poster), exer (exercise-based posters), and both. The interventions appear to influence sales, as suggested by changes in the lines' trajectories post-intervention. The plots are faceted by site, allowing for a comparative view of sales patterns across different locations.
+
+```{r sales-time-series-plot, fig.cap="This plot illustrates the daily sales volumes of zero-calorie (in green) and sugary (in blue) beverages across three hospital sites over 30 weeks. The dashed and their corresponding shorthand labels mark the interventions to allow for visual assessment of their impact on beverage sales trends.", echo=FALSE, fig.height=3, fig.width=6, fig.pos="H"}
+plot_sales_time_series(beverage_sales)
+```
+
+
+For more visual exploration see appendix.
+
+## 4. Formal Analysis
```

Comment:
* The subsections below did not mention any weaknesses and assumptions in the models suggested.
* Techniques and procedures that apply to all three models such as Bonferroni correction and the exclusion of wash periods should be addressed in a separate subsection to avoid repetitiveness.